app/models/attribute.rb

```diff
@@ -83,8 +85,31 @@ def as_txt(attributes, version:, locale: "en")
       ].join("\n")
     end
 
+    def sort_values(attribute_friendly_id, values, locale: "en")
+      return ValueSorter.sort_by_localized_name(
+        values,
+        locale: locale,
+      ) unless @custom_attribute_value_order_indexes&.dig(attribute_friendly_id)
+
+      ordered_attribute_values = @custom_attribute_value_order_indexes[attribute_friendly_id]
+      values.sort_by do |value|
+        ordered_attribute_values[value.friendly_id]
+      end
+    end
+
     private
 
+    def store_custom_attribute_value_order_indexes(data_yml_base_attributes)
```

Comment:
This feels like it is done at the wrong level. Everytime we want positional data we'll have to import custom information from data/attributes.yml

Could we make it so there is a `position` argument on values and when we seed the database set the position argument if it matches one of our special sort keys?